web/.env.production

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+VUE_APP_SERVER_ADDRESS=https://34.227.42.195
```

Comment:
I think what you want to do rather than hard code the domain in the app is to write a [netlify proxy rule](https://docs.netlify.com/routing/redirects/rewrites-proxies/#proxy-to-another-service) to proxy requests to that address.  That will also avoid the need to set up CORS.